layouts/shortcodes/docs/instrumentation/manual-intro.md

```diff
@@ -0,0 +1,16 @@
+[Manual instrumentation](/docs/concepts/instrumentation/manual/) is the act
+of adding observability code to your application yourself.
+
+If you're instrumenting an application, you need to use the OpenTelemetry SDK
+for your language. You'll then use the SDK to initialize OpenTelemetry and the
+API to instrument your code. This will emit telemetry from your app and any
+library you installed that also comes with instrumentation.
+
+If you're instrumenting a library, you must only install the OpenTelemetry API
+package for your language. This means that your library will only emit telemetry
+when an application that uses the OpenTelemetry SDK uses it. Otherwise, your
+library will just behave as before. For more on instrumenting libraries, read
+[these guidelines](/docs/concepts/instrumentation/libraries/).
+
```

Comment:
I'd also recommend including a sentence here about "manual instrumentation does not necessarily mean creating the span/metrics by yourself. One may leverage instrumentation libraries (add link to it)"..

Reply:
I would disagree with that. At the risk of making this a bikeshedding moment, I think instrumentation libraries are much more in the realm of automatic instrumentation than manual instrumentation. In many cases it's literally just one line of code and then you get a bunch of data.

Reply:
We all have different opinions, but its already clarified here:
https://opentelemetry.io/docs/concepts/instrumentation/automatic/

Reply:
Well, the definition of manual instrumentation we have here is pretty clear: https://opentelemetry.io/docs/specs/otel/glossary/#manual-instrumentation

And given packages like node's autoinstrumentations-node package, I still stand by the statement that an instrumentation library is more close to automatic instrumentation than manual instrumentation.

Reply:
This is an [endless](https://github.com/open-telemetry/opentelemetry.io/issues/1689) [discussion](https://github.com/open-telemetry/opentelemetry-specification/issues/2866) and end-user's (aka readers of the documentation) should not be bothered with.

I agree with what @cijothomas said in essence: if an end user is doing manual instrumentation they will make use of 3rd party libraries and they don't want to instrument them themselves, so they either hope that this library is natively instrumented (yeah!) OR they need to make use of an instrumentation library.

If this is no "manual instrumentation" or not, does not really matter for the end user, what matters is that they can and should make use of those to get a better instrumentation experience.

So, I hope we can solve this by wording, e.g.: 

> if you **do** manually instrument your code, you don't have to create all telemetry by yourself: your dependencies either provide native instrumentation for you, or you can make use of instrumentation libraries. To learn more about instrumentation libraries, read [this]. 

My goal is to choose a wording that neither implies the one or the other stance. 

P.S.: If you like to continue the bikesheeding, I refer to the existing discussions on this topic I linked above and, for `autoinstrumentation-node` package additionally, see https://github.com/open-telemetry/opentelemetry-js/issues/3298 (although things changed because this package now actually contains the "auto instrumentation": https://opentelemetry.io/docs/instrumentation/js/automatic/

Reply:
I don't think that's a good paragraph to add/modify. In our own definition we make it very clear that manual instrumentation means using the API to instrument yourself.

The confusion around "automatic" that other maintainers have self-inflicted is just rearing its head again here, because we have components that do not require the use of the API to instrument, and yet they're somehow not considered "automatic instrumentation" or something close to that. It's just weird. You either get components that create data for you or you have to create that data yourself.

Reply:
> I don't think that's a good paragraph to add/modify. In our own definition we make it very clear that manual instrumentation means using the API to instrument yourself.

I actually think the opposite is true and adding this sentence is valuable to end-users, since this question comes up lately often like "can I do manual instrumentation but still use instrumentation library X for my dependency".

Again, that's why I suggested we phrase it in a way that does not suggest that Instrumentation Libraries are manual instrumentation, they can supplement it (if this is the right English word) or you can combine manual instrumentation and instrumentation libraries like you can combine manual instrumentation with the javaagent in java's case (and the same is true for the equivalent in most other languages (actively avoiding the A* word here))

If we don't find consensus here, let's create a follow-up PR to discuss that and try to get this one closed.

Reply:
That bring us into the bikeshed about agents, though 🙂 

I won't die on this hill, for the sake of this PR, but I fear deeply that we're inching towards a path of "technically correct in the way the maintainers reason about things" rather than "what end-users typically care about". Perhaps I'm wrong and people typically see adding an instrumentation library as some form of manual instrumentation, in which case I would propose that we change that definition instead.

Reply:
> I won't die on this hill, for the sake of this PR,

Me neither, that's why I propose we merge this PR for now and create a follow-up one for the other discussion. If this requires a change of definition, then let's see if we re-vive that battle on the spec 😆